app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,32 @@
+/* universal */
+body {
+    padding-top: 60px;
```

Comment:
use 1 tab = 2 spaces, plz

Reply:
![image](https://user-images.githubusercontent.com/68294917/90480582-06c7f100-e163-11ea-84df-18c62c424e6a.png)
like this?